src/System.Private.ServiceModel/src/System/ServiceModel/Description/CustomAttributeProvider.cs

```diff
@@ -83,7 +84,10 @@ public object[] GetCustomAttributes(Type attributeType, bool inherit)
                 case AttributeProviderType.MemberInfo:
                     return this.MemberInfo.GetCustomAttributes(attributeType, inherit).ToArray();
                 case AttributeProviderType.ParameterInfo:
-                    return this.ParameterInfo.GetCustomAttributes(attributeType, inherit).ToArray();
+                    //GetCustomAttributes could return null instead of empty collection for unknown reason here, workaround the issue by explicitly checking the null
+                    IEnumerable<Attribute> customAttributes = null;
```

Comment:
Ron found this is a known issue to system.reflection team. I'll update the comment.